build_info/libpango1.0-0.control

```diff
@@ -0,0 +1,22 @@
+Package: libpango-1.0-0
+Version: @DEB_PANGO_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Depends: fontconfig, libfribidi0, libglib2.0-0, libharfbuzz0b
```

Comment:
Do versioned depends.

Reply:
We don't have any depends for this file that is low enough to conflict